tests/test_part_two.py

```diff
@@ -0,0 +1,42 @@
+from source.part_two import categorize_strand as categorize_encoded_strand
```

Comment:
Since this file is separate from part_one, can we skip the function renaming here (and the calls)?
```suggestion
from source.part_two import categorize_strand
```